### PR TITLE
Task 33

### PR DESCRIPTION
**Pull request by miyabitanimchi**
Good morning @pmvald . Problem solved, I just pushed again and this time they asked me for my username and my personal access token. I input them and could push.

### CODE REVIEW

File: -sass/styles/_footers.m.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.s-footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: $spacer;
+  background: $light-grey;
+  font-size: $font-size-sm;
```
Comment (pmvald) [inline -sass/styles/_footers.m.scss:8]: Good

File: -sass/styles/_footers.m.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  &__mortgage-licence {
+    margin-bottom: 0.5 * $spacer;
+    text-align: justify;
+  }
```
Comment (pmvald) [inline -sass/styles/_footers.m.scss:13]: No need to define a class for everything! These styles can easily be replaced by layer 0 classes "mb-h text-center"

File: -sass/styles/_footers.m.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  &__real-estate-licence {
+    margin-bottom: 1.5 * $spacer;
+    text-align: justify;
+  }
```
Comment (pmvald) [inline -sass/styles/_footers.m.scss:18]: same as above

File: -sass/styles/_footers.m.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+  &__contact-info {
```
Comment (pmvald) [inline -sass/styles/_footers.m.scss:20]: you already defined one flexbox above, isn't it enough? Try to minimize the number of classes ... I believe just one flexbox with justify-content:center is enough

File: -sass/styles/_footers.m.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    display: flex;
+    margin-bottom: 1.5 * $spacer;
+
+    .bi-envelope {
```
Comment (pmvald) [inline -sass/styles/_footers.m.scss:24]: Never include a specific icon in a layer 1 style, which is meant to be a general purpose class (what if someone uses another icon name?) - just use the tag `i` for icon -- and BTW, we don't even need this style - just add "me-q" to the html icon

File: -sass/styles/_footers.m.scss [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      margin-right: 0.2 * $spacer;
+    }
+
+    &__contact-info__email {
+      margin-right: 0.5 * $spacer;
```
Comment (pmvald) [inline -sass/styles/_footers.m.scss:29]: same as above

File: -sass/styles/_footers.m.scss [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    .bi-headset {
+      margin-left: 0.5 * $spacer;
+      margin-right: 0.2 * $spacer;
+    }
```
Comment (pmvald) [inline -sass/styles/_footers.m.scss:35]: same as above

File: -sass/styles/_footers.m.scss [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  &__copyright {
+    margin-bottom: $spacer;
+
+    &__copyrights {
+      margin-bottom: 0.5 * $spacer;
+    }
+
+    &__term-policy {
```
Comment (pmvald) [inline -sass/styles/_footers.m.scss:45]: why so many classes? if you want margin/padding just use bootstrap helpers that we make (m-q, m-h, m-1, p-2, etc.)

**Review by pmvald — CHANGES_REQUESTED** [review]
@miyabitanimchi I made code review. You defined too many classes! This task can be done by just 3 or 4 elements at most ...

**Comment by miyabitanimchi** [comment]
Hi @pmvald, thank you for your review. I modified the files so please check. <br/>
I have a question but what do "h" and "q" mean in bootstrap helpers? (like me-q's "q" and mb-h's "h").

**Review by pmvald — APPROVED** [review]
@miyabitanimchi I made changes. That's how it should be! Please review my changes. For mobile pages it is better to show the icons and text in stack layout than horizontal (like using d-flex) because the screen is narrow and things may squeeze. Also, secondary important text should become grey as I did.

**Comment by pmvald** [comment]
> Hi @pmvald, thank you for your review. I modified the files so please check.
> 
> I have a question but what do "h" and "q" mean in bootstrap helpers? (like me-q's "q" and mb-h's "h").

@miyabitanimchi These are our own inventions!!! Bootstrap only has numbers: m-1, m-2, etc. and they have pretty random values. We made them more organized. q mean quarter (0.25) and h means half (0.5). So m-1q means margin `1.25*$spacer`, p-hq means padding `0.75*$spacer`. Basically by these combination we can create all practical values for spaces

**Pull request merged**